### ISSUE TITLE
BUG:  Test failure for test_draw_busco_plots_for_render

### ISSUE THREAD

**Issue by ebolyen** [bokulich-lab/q2-moshpit#146]
**Describe the bug**

Trying to get the prepare set up so that we can have a dev environment for the shotgun distro. I'm seeing some issues with the Vega test for busco. It looks like the JSON has shifted a little bit and it's causing the tests to fail with `altair=5.2.0`. 

```
FAILED test-env/lib/python3.8/site-packages/q2_moshpit/busco/tests/test_utils.py::TestBUSCO::test_draw_busco_plots_for_render - AssertionError: {'autosize': {'resize': True, 'type': 'pad'[19667 chars]0'}]} != {'config': {'autosize': {'resize': True}, '[11809 chars]': 3}
```

I suspect this PR is related: https://github.com/bokulich-lab/q2-moshpit/pull/142
Notably, we've got 
```
+  'spacing': 3}
```
in the assertion failure (although I can't tell from which direction the spacing is added or missing), but that was changed in the above PR.

**To Reproduce**
See https://github.com/qiime2/distributions/actions/runs/8237435747/job/22532811036

**Comment by ebolyen** [comment]
For a limited window, we can download the built conda packages:
https://github.com/qiime2/distributions/actions/runs/8237435747/artifacts/1316571672
And environment file which is under test:
https://github.com/qiime2/distributions/actions/runs/8237435747/artifacts/1316571675
(you will need to rename the top channel to match your extracted conda package channel location)

**Comment by misialq** [comment]
Hey @ebolyen, I'm guessing this got "fixed" after the revert in #147? Should we keep this open or we close and re-open when it re-surfaces again in the next BUSCO PRs?

**Comment by ebolyen** [comment]
CI should catch it this time around!
